installers/vmware/kickstart-esxi.go

```diff
@@ -383,6 +384,9 @@ func rootpw(j job.Job) string {
 // We do not support anything other than ESXi 6.5 and above (os slug "vmware_esxi_6_5", "vmware_esxi_6_7", "vmware_esxi_7_0" etc)
 // full list of drive settings is listed https://packet.atlassian.net/browse/SWE-2385
 func determineDisk(j job.Job) string {
+	if j.BootDriveHint() != "" && strings.HasPrefix(j.PlanSlug(), "s") {
```

Comment:
Why do we want to limit it to just the `s` plans? If there's a boot drive hint we should honor it, if it shouldn't be there then we should update metadata to not have it at all.

Reply:
To limit blast radius at initial roll out. This is also the plan that is most in need of this feature.

Reply:
how about a comment then, this way if you forget to come back this wont stay around forever :D

Reply:
Will do. There are instructions in the PR description, but I'll add a short note just above that line in the code.

Reply:
Better?

Reply:
yep, thanks!